### PR TITLE
[MBL-17283] [Student][Teacher] Create edit delete calendar event

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: 
- functionality should match with the web 
- designs are in Figma

refs: MBL-17283
affects: Student, Teacher
release note: Create, edit, delete calendar events.

## Checklist

- [x] Follow-up e2e test ticket created or not needed
- [ ] A11y checked
- [x] Approve from product

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/calendarevent/createupdate/composables/CustomFrequencyScreen.kt [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+                OutlinedTextField(
+                    value = uiState.quantity.takeIf { it > 0 }?.toString().orEmpty(),
+                    onValueChange = {
+                        actionHandler(CreateUpdateEventAction.UpdateCustomFrequencyQuantity(it.toIntOrNull().orDefault()))
+                    },
+                    modifier = Modifier
+                        .weight(1f)
+                        .padding(horizontal = 16.dp),
+                    singleLine = true,
+                    keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.NumberPassword),
```
Comment (tamaskozmer) [inline libs/pandautils/src/main/java/com/instructure/pandautils/features/calendarevent/createupdate/composables/CustomFrequencyScreen.kt:179]: I would change this to `KeyboardType.Number`.

Comment (kristofnemere) [reply libs/pandautils/src/main/java/com/instructure/pandautils/features/calendarevent/createupdate/composables/CustomFrequencyScreen.kt:179]: That had . in it

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Pull request merged**